classloader-leak-test-framework/src/main/java/se/jiderhamn/classloader/leak/JUnitClassloaderRunner.java

```diff
@@ -133,6 +141,11 @@ public void evaluate() throws Throwable {
         
         forceGc(3);
 
+        if (assumptionFailed) {
```

Comment:
Do we want to do this before forcing garbage collection 3 times, for better performance?

Reply:
Hmm, makes sense.